mets_mods2tei/api/tei.py

```diff
@@ -724,8 +724,48 @@ def add_div_structure(self, div, pages=None):
             elif sub_div.get_TYPE() == "title_page":
                 self.__add_div(entry_point, sub_div, 1, "titlePage")
             else:
+                # FIXME: if title_page gets preceded by figure/preface/contents/..., they *all* will end up in body
                 entry_point = body
                 self.__add_div(entry_point, sub_div, 1)
+            # FIXME: add more structural mappings from METS-Anwendungsprofil (DFG Strukturdatenset) to TEI-P5 tagset (DTAbf)
+            # ...for example:
+            # contents → contents
+            # corrigenda → corrigenda
+            # dedication → dedication
+            # index → index
+            # imprint → imprint
+            # ? → imprimatur
+            # priviledges? → copyright
+            # provenance → ?
+            # ? → appendix
+            # ? → advertisement
+            # preface → preface
+            # ? → postface
+            # chapter → chapter
+            # letter → letter
+            # verse → poem
+            # ? → diaryEntry
+            # ? → recipe
+            # ? → scene
+            # ? → act
+            # ? → frontispiece
+            # ? → bibliography
+            # list_illustrations? → figures
+            # ? → abbreviations
+            # ? → edition
+            # cover → ?
+            # cover_front → ?
+            # cover_back → ?
+            # table → ?
+            # manuscript → ?
+            # illustration → ?
+            # section → ?
+            # article → ?
+            # issue → ?
+            # day → ?
+            # month → ?
+            # volume → ?
+            # year → ?
```

Comment:
Could this be put in an issue por favor? I.e as a feature request?

Reply:
Preferably, yes – but GH only allows issue-referencing code which is on master. (I'll open a new issue of wider scope as soon as we cleared some of the fog.)